src/css/vjs.scss

```diff
@@ -1,3 +1 @@
-$icon-codepoints: true;
```

Comment:
Removing this file entirely would technically be a breaking change, even removing `icon-codepoints` could be questionable but I don't think many people have been using it.